static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js

```diff
@@ -157,6 +157,12 @@ const getImproveSizeParam = (slotId: string): PrebidImproveSizeParam => {
         : {};
 };
 
+const getXaxisPlacementId = (sizes: PrebidSize[]): number => {
+    if (contains(sizes, [970, 250])) return 13218365;
+    if (contains(sizes, [300, 600])) return 12984524;
+    return 13218372;
```

Comment:
What are these IDs? Maybe they deserve a descriptive name?

Reply:
Do you mean the dimensions?

Reply:
No, the placement IDs.I suppose you could just figure it out from the function name.

Reply:
Yes they don't really have any meaning.  I could call them `xaxisBillboardPlacementId` or something but I think it would become even more unreadable.  As long as they're not repeated anywhere it should be OK.